test/e2etest/installer_test.go

```diff
@@ -25,12 +25,24 @@ func TestInstaller(t *testing.T) {
 	// Install the environment
 	runInstallScript(testConfig.ConfigPath)
 
-	// Tests come here
+	// Test the PAUSE status
+	pauseServer(t, testConfig)
+	assertStatusEndpoint(t, testConfig, "PAUSED")
+
+	// Test the RUNNING status
+	resumeServer(t, testConfig)
+	assertStatusEndpoint(t, testConfig, "RUNNING")
 
 	// Uninstall and cleanup the environment
 	runUninstallScript(testConfig.ConfigPath)
 }
 
+func assertStatusEndpoint(t *testing.T, testConfig TestConfig, expectedStatus string) {
+	statusUrl := "rest/api/latest/status"
+	url := fmt.Sprintf("https://%s.%s.%s/%s", product, testConfig.EnvironmentName, domain, statusUrl)
+	content := fmt.Sprintf("%s", GetPageContent(t, url))
```

Comment:
`fmt.Sprintf` is probably unnecessary here. Something like this might be sufficient: 
```
        content := GetPageContent(t, url)
	assert.Contains(t, string(content), "<title>All projects - Atlassian Bamboo</title>")
	assert.Contains(t, string(content), "totalRecords: 1")
```

Reply:
This is needed as `GetPageContent` returns `byte[]` and we need `string`. This actually converts `byte[]` to `string`.

Reply:
Doesn't `string(content)` also convert `[]byte` to `string`? 
Just curious, from Google I found most answers use the above way I copied from our repo, i.e. `string(content)`. https://yourbasic.org/golang/convert-string-to-byte-slice/ 
Did you choose to use `fmt.Sprintf` because it's more efficient? I guess both are workable, and probably doesn't matter so much in our case given we're not too worried about performance. Just trying to close the gap here if you choose `fmt.Sprintf` for a reason.

Reply:
Generally in type conversion to string the best practise is using string builder. This is specially efficient when we creating the string accumulatively to avoid creating multiple short life objects. In our case we convert just once but still better use the best practice.

Reply:
Awesome. Good to know~ 🥇